source/_lovelace/button.markdown

```diff
@@ -23,53 +48,53 @@ type:
   type: string
 entity:
   required: false
-  description: Home Assistant entity ID.
+  description: The entity the card interacts with, for example `light.living_room`.
   type: string
 name:
   required: false
-  description: Overwrites friendly name.
+  description: The button name.
   type: string
-  default: Name of Entity
+  default: Entity name
 icon:
   required: false
-  description: Icon that will be be used to overwrite the entity picture or entity icon.
+  description: The icon shown on the card.
   type: string
   default: Entity Domain Icon
 show_name:
   required: false
-  description: Show name.
+  description: If true, the button name is shown on the card.
```

Comment:
We should flip the logic of the boolean descriptions. since its default true. Let them know what happens if its false

Same for show_icon